src/components/icons/arrowRight.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+
+const ArrowRightTeal = (props) => (
+  <svg {...props}>
+    <path
+      stroke="#3696B9"
+      strokeWidth="1.8"
+      d="M13 10l6 6-6 6"
+      fill="none"
+      strokeLinecap="round"
+    />
+  </svg>
+)
+
+export default ArrowRightTeal
```

Comment:
we try to have as little svgs as possible. if possible, can you use the existing arrow and use css to style the text and rotation? something like...
```
<ArrowDownM className="text-teal rotate-180 transform" />
```